benchmarking/frameworks/generic/generic.py

```diff
@@ -50,7 +55,16 @@ def runBenchmark(self, info, benchmark, platform):
         if model is not None and "cached_modes" in model:
             platform.delFilesFromPlatform(cached_models)
         platform.delFilesFromPlatform(program)
-        if shared_libs is not None:
-            platform.delFilesFromPlatform(shared_libs)
 
         return output, output_files
+
+    def _updateModelPath(self, model, commands):
+        _args = commands.split()
+        init_net = model["files"]["init"]["filename"]
```

Comment:
It may not work with MetaNetDef format. Can we check whether the net exists before accessing the content?